### DISCUSSION TITLE
Login multiple times

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5300] [General]
Hi,

I have a problem with the admin menu. When I login to the admin menu and for example I want to go to the page/menu admin section  then E107 present the login to me dialogue again. If I put in my login data then it present again the login dialogue. Sometimes 4times on a row.

What is wrong? Anyone? It is now become to the point I can hardly do anything in the admin menu. Any help is appreciated.

Kind regards,
Henk - www.ph5hp.nl

**Comment by tgtje** [comment]
@PH5HP maybe unrelated ??  but should not occur

due to the state using both httpS link (.   php5 . nl  and http link (www . php5 . nl) maybe the eu- cookie gets corrupted as call for admin goes to httpS and other to http....  

 (dutch : oftewel : is je site nou https of http ?, dat maakt wel verschil, zeker ook voor browsers) .

**Reply by PH5HP** [reply]
Hi tgtje,

My site is https://

I found out that I think has to do with 'Flood Timeout' which was on 10 (seconds). I increased it to 30 (seconds) and the problem seems to be gone now.

![image](https://github.com/user-attachments/assets/45bad9ca-ff6c-461b-a620-99d0591130b3)

Kind regards,
Henk - ph5hp.nl

**Reply by tgtje** [reply]
@PH5HP  please check ;-)
![bnRE9mE4h8](https://github.com/user-attachments/assets/ba564798-32ab-4f58-9816-9d00acbcbfba)

iff https is running and still no go : it's the use of http mixed with https links (hardcoded cq absolute/full).
To work proper with certificates, all links have to be httpS when such paths exist.

**Reply by PH5HP** [reply]
Hi tgtje,

Thank you for trying to help me. Problem solved with the solution I described above. I think my links are all https://. My provider says that it is normal that a website can be accessed also with http:// if https certificate is valid.

Kind regards,
Henk - ph5hp.nl

**Reply by tgtje** [reply]
Ok, if  both protocols are accepted by the server (likely 2 ports in use.)  ;-)
